agency/README.md

```diff
@@ -261,19 +244,16 @@ For `reason_code`, options are:
 * `rebalancing`
 * `maintenance`
 
-### message
-
-For 'message', options are:
-
-* `200: OK`
-* `201: Created`
-* `202: Accepted`
-* `203: Added`
-* `204: Removed`
-* `210: Warning: vehicle used in this trip has not been properly registered`
-* `305: Error: vehicle is already registered`
-* `306: Error: vehicle registration cannot be found`
-* `310: Error: vehicle is not properly registered`
-* `311: Error: duplicate registration found, please use a different unique_id or update existing unique_id status using the update-vehicle-status endpoint`
-* `315: Error: vehicle is not active`
-* `320: Error: vehicle trip has not been properly started`
+## Responses
+
+RESTful responses are returned for all endpoints, namely:
+
+* **200:** OK, operation successful. For `PUT` operations, object existed and has been updated.
+* **201:** For `PUT` operations, new object created
+* **400:** Malformed request.
+    In this case, the answer presents the list of errors as an `application/json` body;
+    Each error in the list is a string in plain english.
```

Comment:
Ça peut être pratique d'avoir un truc un peu plus structuré avec au moins un dict sur le nom des champs en erreur.

Reply:
I gave 2 examples of simple and clear error response body in previous issue : https://github.com/CityOfLosAngeles/mobility-data-specification/issues/127

Was closed as dupe of https://github.com/CityOfLosAngeles/mobility-data-specification/issues/105

From my experience, returning a list of errors is rarely usefull, but maybe you have an other pov ?